src/components/Table/utils/RowState.ts

```diff
@@ -234,6 +307,28 @@ export class RowState {
     return typeof this.row.pin === "string" || (!!this.row.pin && this.row.pin.filter !== true);
   }
 
+  // mobx will cache this getter for us
+  public get api(): GridRowApi<R> {
+    const rs = this;
+    // Copy the GridTableApi + the getVisibleChildren GridRowApi method
+    return {
+      ...this.states.table.api,
```

Comment:
**thought (non-blocking):** Maybe worth adding a comment here to tie it together that the `this` in the returned table state API methods is going to be the TableState instance due to the binding above, and not the RowState instance.